docs/manuals/userman/gui/pages/partial/index.md

```diff
@@ -59,7 +124,7 @@ gui.run()
 Here, `link_partial` is declared with an empty string. This partial will be dynamically 
```

Comment:
What is this about?

Reply:
I don't understand your comment

Reply:
What is `link_partial`? A class?
If it's a variable, it should be \*link_partial\*. And where is it declared or defined?

Reply:
It is declared just above. I can put the line numbers. I have put it between `*`